modules/router-store/migrations/migration.json

```
@@ -20,6 +20,11 @@
       "description": "The road to v14",
       "version": "14-beta",
       "factory": "./14_0_0/index"
+    },
+    "ngrx-router-store-migration-05": {
+      "description": "The road to v15.1.1",
+      "version": "15.1.1",
+      "factory": "./15_1_1/index"
```

I think the upcoming version should be `15.2.0` because of the new features.

```suggestion
      "description": "The road to v15.2.0",
      "version": "15.2.0",
      "factory": "./15_2_0/index"
```